src/pages/postgraphile/enums.md

```diff
@@ -84,6 +84,63 @@ comment on table animal_type is E'@enum\n@enumName TypeOfAnimal';
 
 The name must conform to the GraphQL identifier restrictions.
 
+#### Functions returning table enums
```

Comment:
@tsnobip Can you update this to cover usage for function arguments being enums too? Here's a suggested title:

```suggestion
#### Functions and table enums
```